src/sound/ScratchSoundPlayer.as

```diff
@@ -70,43 +70,72 @@ public class ScratchSoundPlayer {
 	public function ScratchSoundPlayer(wavFileData:ByteArray) {
 		getSample = getSample16Uncompressed;
 		if (wavFileData != null) {
-			var info:* = WAVFile.decode(wavFileData);
-			soundData = wavFileData;
-			startOffset = info.sampleDataStart;
-			endOffset = startOffset + info.sampleDataSize;
-			stepSize = info.samplesPerSecond / 44100.0;
-			if (info.encoding == 17) {
-				adpcmBlockSize = info.adpcmBlockSize;
-				getSample = getSampleADPCM;
-			} else {
-				if (info.bitsPerSample == 8) getSample = getSample8Uncompressed;
-				if (info.bitsPerSample == 16) getSample = getSample16Uncompressed;
+			try {
+				var info:* = WAVFile.decode(wavFileData);
+				soundData = wavFileData;
+				startOffset = info.sampleDataStart;
+				endOffset = startOffset + info.sampleDataSize;
+				stepSize = info.samplesPerSecond / 44100.0;
+				if (info.encoding == 17) {
+					adpcmBlockSize = info.adpcmBlockSize;
+					getSample = getSampleADPCM;
+				} else {
+					if (info.bitsPerSample == 8) getSample = getSample8Uncompressed;
+					if (info.bitsPerSample == 16) getSample = getSample16Uncompressed;
+				}
 			}
+			catch (e:*) {}
```

Comment:
I added the try/catch because of an exception I saw in Sentry about the waveFileData being length zero. Now it's logging just in case there are other exceptions.